tests/analysis.spec.ts

```diff
@@ -327,5 +346,30 @@ describe('Functional test of analysis', () => {
         }),
       );
     });
+
+    // TODO: this test is blocking the code-client and cli uploads from moving forward
```

Comment:
enh: This should be an actual explanation of what the problem is - the issue is that the flow is not rolled out yet and thus the test can't succeed.